src/openrct2/GameState.h

```diff
@@ -26,6 +26,22 @@ namespace OpenRCT2
 {
     class Park;
 
+    enum class StaffSpeedCheat
+    {
+        None,
+        Frozen,
+        Fast,
+    };
+
+    struct CheatsState
+    {
+        bool AllowTrackPlaceInvalidHeights;
+        bool AllowRegularPathAsQueue;
+        bool AllowSpecialColourSchemes;
+        bool MakeAllDestructible;
+        StaffSpeedCheat SelectedStaffSpeed;
+    };
```

Comment:
Wouldn't it be better to have that in Cheats.h?

Reply:
Aren't we going to end up with a god header that forces us to recompile everything under lots of situations?

Reply:
I mean the game state carries a lot of things so it won't really matter where you put the structs as this header is included in a lot of places, I rather have it organized and ideally we keep types and logic separated anyway.